lib/ddr/models/indexing.rb

```diff
@@ -41,6 +41,9 @@ def index_fields
           fields[MEDIA_MAJOR_TYPE] = content_major_type
           fields[MEDIA_SUB_TYPE] = content_sub_type
         end
+        if is_a? Component
+          fields[COLLECTION_URI] = collection.internal_uri unless collection.nil?
```

Comment:
Nothing wrong, but I wonder if it's slightly better for reusability to define `#collection_uri` on the Component model?